rabbitmq/tests/test_openmetrics.py

```diff
@@ -22,6 +22,27 @@ def test_aggregated_endpoint(aggregator, dd_run_check, mock_http_response):
 
     expected_metrics = (
         [
+            dict(
```

Comment:
This is very thorough! You don't have to assert every metric individually. You can use
`aggregator.assert_metrics_using_metadata` to assert the metric types and `aggregator.assert_all_metrics_covered` to assert all expected metrics.

For the the metrics with tags, you can assert those individually

Reply:
https://github.com/DataDog/integrations-core/pull/13693

Let me know what you think. This refactored doesn't assert for the values or other tags on the other metrics, but it should be fine.